cohortextractor/databricks_backend.py

```diff
@@ -3,11 +3,20 @@
 
 class DatabricksBackend:
     def __init__(self, database_url, covariate_definitions, temporary_database=None):
-        self.next_temp_table_id = 1
-        self.session = self.get_spark_session(database_url)
+        self.next_temp_table_id = 0
+        self.session = self.create_spark_session()
         self.dataframes = self.build_dataframes(covariate_definitions)
 
+    def create_spark_session(self):
+        conf = SparkConf().set(
+            "spark.driver.extraClassPath", "sqljdbc_9.2/mssql-jdbc-9.2.1.jre11.jar"
```

Comment:
Is this relative to the cwd? I wonder if you need to do something with `__file__` here if this is to work when cohortextractor is invoked inside the `/workspace` directory.

Reply:
Quite probably.  Will fix.